ement.el

```diff
@@ -151,6 +151,12 @@ Alist mapping user IDs to a list of room aliases/IDs to open buffers for."
   :type '(alist :key-type (string :tag "Local user ID")
                 :value-type (repeat (string :tag "Room alias/ID"))))
 
+(defcustom ement-read-reaction #'ement-read-reaction
+  "Function that prompts the user for reaction string.
+The function is called with no arguments and should return a
+string to be used as the reaction."
+  :type 'function)
```

Comment:
For completeness, please add the default function in the option's type so the user can always select it.  See other defcustoms in the project for how that is done.